test/cypress/support/index.js

```diff
@@ -16,6 +16,12 @@
 import './commands.js'
 import './output-logs.js'
 
+before(function () {
+  cy.clearCookies()
+  cy.clearLocalStorage()
+  indexedDB.deleteDatabase('Group Income')
+})
+
```

Comment:
I checked and it seems 'necessary'. If this isn't here, then if a test fails (say, because the user state was corrupted, as it happened as I tried different approaches here), then the issue carries over. IMO, Cypress should start each test with a blank / fresh state, but it doesn't. By having this, then an error in one test doesn't affect the other tests, which I think is more sensible.

Reply:
The necessity of this is however something that can be handled / discussed in a separate issue, as it's not technically required for this particular PR